Subject: Document Debian's decision to disable modelines by default
 Modelines have historically been a source of vulnerabilities in Vim.
 As long it remains a "blacklist suspected/proven dangerous options"
 instead of a "whitelist allowed options" piece of functionality,
 Debian's system-wide vimrc will maintain this setting.  As such, the
 documentation needs to be updated to reflect the induced behavior.
Forwarded: not-needed
Author: James Vega <jamessan@debian.org>
Bug-Debian: http://bugs.debian.org/472522
Bug-Ubuntu: https://launchpad.net/bugs/130531

--- a/runtime/doc/options.txt
+++ b/runtime/doc/options.txt
@@ -4969,7 +4969,7 @@
 
 				   *'modeline'* *'ml'* *'nomodeline'* *'noml'*
 'modeline' 'ml'		boolean	(Vim default: on (off for root),
-				 Vi default: off)
+				 Debian: off, Vi default: off)
 			local to buffer
 						*'modelines'* *'mls'*
 'modelines' 'mls'	number	(default 5)
